### DISCUSSION TITLE
Aiuto per laser da 20w..Help for a 20w laser

### DISCUSSION THREAD

**Discussion by thecuress** [arkypita/LaserGRBL#1018] [Q&A]
Ciao ...Ho acquisto su un sito un laser da 20w marca NEJE.....come sofware uso laserGRBL e lightburn....Per i primi due o tre giorni sono riuscito a tagliare legno spesso 3mm...con tre quattro passaggi...Riprendendo successivamente il lavoro..,questo non e piu riuscito a tagliare,premetto che non ho toccato ne fuoco e ne altri parametri...riesce solo a incidere forse 1 mm.Pulito anche la lente ,ma senza successo...qualche soluzione (..se esiste)..grazie mille 

Hi ... I purchase a 20w NEJE brand on a site ..... as a software I use laser GRBL and lightburn .... For the first two or three days I managed to cut 3mm thick wood ... with three four steps ... Resuming the work later .., this was no longer able to cut, I state that I have not touched the fire and other parameters ... only manages to affect maybe 1 mm. Also cleaned the lens, but without success. ..some solution (..if it exists) .. thank you very much...sorry for bad translation ..i use google translate

**Comment by gmmanonymus111** [comment]
It is possible your laser diode died. or your lens got smokey (since you already cleaned it, the first one is assumed). NEJE has 6 months warranty for their laser modules AFAIK, so your best bet is contact the seller for a replacement.
Also, before contacting the seller, make sure you ACCIDENTALLY did not change the laser power or engraving speed.
